Assets/Scripts/DataController.cs

```diff
@@ -120,18 +123,10 @@ void UpdateOnSceneLoad(bool showHorizon, float ts, float rad, float mag, bool sh
         {
             StarComponent starComponent = allStarComponents[i].gameObject.GetComponent<StarComponent>();
             Utils.SetObjectColor(allStarComponents[i].gameObject, consColor ? starComponent.constellationColor : Color.white);
-            if (showHorizon)
-            {
-                allStarComponents[i].gameObject.transform.position = starComponent.starData.CalculateHorizonPosition(radius, localSiderialStartTime, 0);
-            }
-            else
-            {
-                allStarComponents[i].gameObject.transform.position = starComponent.starData.CalculateEquitorialPosition(radius);
-            }
-            var magScaleValue = ((starComponent.starData.Mag * -1) + maxMag + 1) * magnitudeScale;
-            Vector3 magScale = new Vector3(1f, 1f, 1f) * magScaleValue;
-            allStarComponents[i].gameObject.transform.localScale = magScale;
```

Comment:
We actually need this calculation of `magScaleValue` and assignment to the transform localScale since the stars scales vary from scene to scene.  Without it the stars remain small if you load `Stars` first or large if you load `Horizon` first.

Reply:
Good catch, I'll get a fix for that.